lib/rules/color-named/index.js

```diff
@@ -49,7 +50,7 @@ const rule = function (expectation, options) {
     const namedColors = Object.keys(namedColorData)
 
     root.walkDecls(decl => {
-      if (propertySets.acceptCustomIdents.has(decl.prop)) {
+      if (propertySets.acceptCustomIdents.has(decl.prop) || !isStandardSyntaxDeclaration(decl)) {
```

Comment:
I pushed a commit up with what seems to produce expected results, though sans `isStandardSyntaxFunction` due to the above.